### PR TITLE
[CLOUDTRUST-5511] Add initial onboarding status on user creation

### PR DESCRIPTION
**Pull request by Xelowak**

### CODE REVIEW

File: internal/business/gln_test.go [hunk 1]
```diff
@@ -60,71 +60,73 @@ func (m *mockLookup) Lookup(gln string) GlnSearchResult {
...
 func TestGln(t *testing.T) {
-	var mockCtrl = gomock.NewController(t)
+	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
-	var anError = errors.New("an error")
-	var gln = "111111111"
-	var mockProviders = []GlnLookupProvider{newMockLookup(1, time.Millisecond*100, nil), newMockLookup(2, time.Millisecond*10, nil),
-		newMockLookup(3, time.Millisecond*10, nil)}
+	anError := errors.New("an error")
```
Comment (fperot74) [inline internal/business/gln_test.go:66]: Has a change in formatting been agreed in the team?

**Review by fperot74 — APPROVED** [review]
Can't review this PR... it is polluated by lots of unexpected formatting changes

**Review by harture — CHANGES_REQUESTED** [review]
Not against applying new formatting but it must be in a separated commit else it cannot be reviewed.
However, we have to agree altogether if we want to apply a new formatting else it will be quickly a nightmare for everyone :) Let's discuss it

**Comment by Xelowak** [comment]
I close this PR and open a new one because of formatting issues

**Pull request closed without merging**